routes/account/login/lib/localPassport.js

```diff
@@ -3,9 +3,9 @@ var passport        = require('passport');
 var LocalStrategy   = require('passport-local').Strategy;
 var modelUser       = require('models/User');
 
-var LocalStrategyField = {
+const LocalStrategyField = {
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).